CedarJava/src/main/java/com/cedarpolicy/model/AuthorizationResponse.java

```diff
@@ -53,4 +53,14 @@ public AuthorizationResponse(
             this.warnings = ImmutableList.copyOf(warnings);
         }
     }
+
+    @Override
```

Comment:
Did you mean to include these new `toString`s?

Reply:
Yes, they are helpful for debugging.  Debugging test failures is currently painful

Reply:
Cool, just making sure it wasn't an accidental inclusion